source/network-security.txt

```diff
@@ -41,7 +41,7 @@ configuration of your {+clusters+}.
 databases.
 
 We recommend using M10+ dedicated {+clusters+} because all {+service+} projects with one or more M10+ dedicated {+clusters+} receive
```

Comment:
```suggestion
We recommend using M10+ dedicated {+clusters+} because all {+service+} projects with one or more M10+ dedicated {+clusters+} receive their own dedicated:
```